packages/neos-ui/src/Containers/LeftSideBar/NodeTreeSearchBar/NodeTreeSearchInput/style.css

```diff
@@ -1,24 +1,18 @@
 .searchInput {
-    flex-grow: 1;
-
     input {
         /* magic number, sorry */
         padding-left: 40px;
         border-radius: 0;
+
+        &:focus {
```

Comment:
That indeed is an "unsolved question" ... almost all textfields become white on focus, but the filter does not, maybe because it's some kind of dropdown / combined search dropdown. We should clean things up later on.

Reply:
Ok, could you please open an issue for this, so we could take care of it later?

Reply:
https://github.com/neos/neos-ui/issues/2456